python/cudf/cudf/utils/gpu_utils.py

```diff
@@ -24,7 +24,11 @@ def validate_setup(check_dask=True):
 
     try:
         gpus_count = getDeviceCount()
-    except CUDARuntimeError:
+    except CUDARuntimeError as e:
+        if e.status == 35:
+            # status = 35 Indicates CUDA driver
+            # is older than CUDA runtime library
+            raise e
```

Comment:
I imagine there's more than just this status that we'd want to raise. Maybe we should create a set of statuses that we don't want to raise on and check `if e.status not in allowed_statuses`?

Reply:
Added a set of status codes which we will want to raise based on the list here: https://docs.nvidia.com/cuda/cuda-runtime-api/group__CUDART__TYPES.html#group__CUDART__TYPES_1g3f51e3575c2178246db0a94a430e0038